src/models/users.js

```diff
@@ -13,9 +13,8 @@ export default (sequelize, DataTypes) => {
     lineManagerId: DataTypes.INTEGER,
     role: DataTypes.ENUM('super_admin', 'travel_admin', 'travel_team_member', 'manager', 'requester'),
     profilePicture: DataTypes.STRING(1234),
-    token: DataTypes.STRING,
-    isVerified: DataTypes.BOOLEAN,
-    lineManagerId: DataTypes.INTEGER
```

Comment:
Why is `lineManagerId` removed? I see it is used by other team members.

Reply:
It was duplicated. You can see `Line 13` and `Line 18`